src/components/oc-client/template.jade

```diff
@@ -3,4 +3,4 @@ script.
   oc.conf = oc.conf || {};
   oc.conf.templates = oc.conf.templates || [];
   oc.conf.templates = oc.conf.templates.concat(!{JSON.stringify(templates)});
-script(src=staticPath+"src/oc-client.min.js", type="text/javascript")
+script(src="https://unpkg.com/oc-client-browser@1.0.0/dist/oc-client.min.js", type="text/javascript")
```

Comment:
I think we should update the oc's build script so that this version (1.0.0) comes from a `require('oc-client-browser/package').version`.